src/views/preview/comment/comment.jsx

```diff
@@ -214,6 +216,7 @@ Comment.propTypes = {
     author: PropTypes.shape({
         id: PropTypes.number,
         image: PropTypes.string,
+        is_staff: PropTypes.bool,
```

Comment:
Even though this is what we use, we probably would want to clean up this name in the API so it was just `isScratchTeam` or `scratchteam`?

Reply:
hmm yeah, although as with `isPublished` vs. `is_published`, I think it gets even more confusing when you start renaming properties...

Reply:
Kinda... this is the job of our models in the API though. There are plenty of other places where we don't blindly pass though database fields.

Reply:
Ok, well let me know what it should be and I'll change it here and in my PR against API https://github.com/LLK/scratch-api/pull/614